modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -3,31 +3,41 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: REFERENCE
-[id="example-remote-write-authentication-settings_{context}"]
-= Example remote write authentication settings
 
-// Set attributes to distinguish between cluster monitoring examples and user workload monitoring examples.
-ifndef::openshift-dedicated,openshift-rosa[]
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="example-remote-write-authentication-settings-cpm_{context}"]
+= Example remote write authentication settings for core platform monitoring
+// end::CPM[]
+
+// tag::UWM[]
+[id="example-remote-write-authentication-settings-uwm_{context}"]
+= Example remote write authentication settings for monitoring of user-defined projects
+// end::UWM[]
+
+// Set attributes to distinguish between cluster monitoring example (core platform monitoring - CPM) and user workload monitoring (UWM) examples
+// tag::CPM[]
 :configmap-name: cluster-monitoring-config
 :namespace-name: openshift-monitoring
-:prometheus-instance: prometheusK8s
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
+:component: prometheusK8s
+// end::CPM[]
+// tag::UWM[]
 :configmap-name: user-workload-monitoring-config
 :namespace-name: openshift-user-workload-monitoring
-:prometheus-instance: prometheus
-endif::openshift-dedicated,openshift-rosa[]
+:component: prometheus
+// end::UWM[]
 
 The following samples show different authentication settings you can use to connect to a remote write endpoint. Each sample also shows how to configure a corresponding `Secret` object that contains authentication credentials and other relevant settings. Each sample configures authentication for use with
-ifndef::openshift-dedicated,openshift-rosa[]
+// tag::CPM[]
 default platform monitoring
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
-monitoring user-defined projects
-endif::openshift-dedicated,openshift-rosa[]
+// end::CPM[]
+// tag::UWM[]
+monitoring for user-defined projects
+// end::UWM[]
 in the `{namespace-name}` namespace.
 
-[id="remote-write-sample-yaml-aws-sigv4_{context}"]
+//commenting this ID out for the build to not break. Will be uncommented after the migration.
+//[id="remote-write-sample-yaml-aws-sigv4_{context}"]
```

Comment:
I'm not sure it's breaking anything by doing this but I think even for a pre-production branch you should probably still have unique IDs for every header. This is because when you don't set an ID, one is automatically generated based on the text in the heading. 

And in this case, you're using the same module twice in the same assembly, so each ID you skip creates an instance where you have duplicated anchors in the assembly. 

If it's not much extra effort for you, I'd rather see you make two dummy ID values based on tags and just plan to delete those whenever you do your final implementation.

Reply:
I see, I did not know about that, thank you for letting me know! I will make that change now.